python/vowpalwabbit/sklearn_vw.py

```diff
@@ -122,112 +124,189 @@ def __init__(self,
-        invert_hash (str): Output human-readable final regressor with feature names.  Computationally expensive.
-        save_resume (bool): save extra state so learning can be resumed later with new data
-        output_feature_regularizer_binary (str): Per feature regularization output file
-        output_feature_regularizer_text (str): Per feature regularization output file, in text
+
+        final_regressor,f : str
+            Final regressor
+        readable_model : str
+            Output human-readable final regressor with numeric features
+        invert_hash : str
+            Output human-readable final regressor with feature names.  Computationally expensive.
+        save_resume : bool
+            save extra state so learning can be resumed later with new data
+        output_feature_regularizer_binary : str
+            Per feature regularization output file
+        output_feature_regularizer_text : str
+            Per feature regularization output file, in text
 
         Multiclass options
-        oaa (int): Use one-against-all multiclass learning with labels
-        ect (int): Use error correcting tournament multiclass learning
-        csoaa (int): Use cost sensitive one-against-all multiclass learning
-        wap (int): Use weighted all pairs multiclass learning
+
+        oaa : integer
+            Use one-against-all multiclass learning with labels
+        ect : integer
+            Use error correcting tournament multiclass learning
+        csoaa : integer
+            Use cost sensitive one-against-all multiclass learning
+        wap : integer
+            Use weighted all pairs multiclass learning
 
         Contextual Bandit Optimization
-        cb (int): Use contextual bandit learning with specified costs
-        cbify (int): Convert multiclass on <k> classes into a contextual bandit problem
+
+        cb : integer
+            Use contextual bandit learning with specified costs
+        cbify : integer
+            Convert multiclass on <k> classes into a contextual bandit problem
 
         Neural Network options
-        nn (int): Use a sigmoidal feed-forward neural network with N hidden units
-        dropout: Train or test sigmoidal feed-forward network using dropout
-        inpass: Train or test sigmoidal feed-forward network with input pass-through
-        multitask: Share hidden layer across all reduced tasks
-        meanfield: Train or test sigmoidal feed-forward network using mean field
+
+        nn : integer
+            Use a sigmoidal feed-forward neural network with N hidden units
+        dropout : Train or test sigmoidal feed-forward network using dropout
+        inpass : Train or test sigmoidal feed-forward network with input pass-through
+        multitask : Share hidden layer across all reduced tasks
+        meanfield : Train or test sigmoidal feed-forward network using mean field
 
         Returns
         -------
+
         (BaseEstimator): Returns self
```

Comment:
I don't think all the returns March the numpy docstrings format.?

Should be:

Returns
-------
int
    Description of anonymous integer return

Reply:
Yes, I have changed that in numpydoc style